lib/src/styled_element.dart

```diff
@@ -25,7 +25,7 @@ class StyledElement {
   bool matchesSelector(String selector) =>
       _node != null && matches(_node, selector);
 
-  Map<String, String> get attributes => _node.attributes.map((key, value) {
+  Map<String, String> get attributes => _node?.attributes?.map((key, value) {
```

Comment:
This would still cause a similar issue because if the `attributes` map is accessed and it is null, then we would crash. If there are no attributes, it should be an empty map, not a null one.

It should be something like `(_node?.attributes ?? Map<String,String>())`